src/rawdog/llm_client.py

```diff
@@ -79,23 +69,28 @@ def get_response(
         self,
         messages: list[dict[str, str]],
     ) -> str:
+        base_url = self.config.get("llm_base_url")
+        model = self.config.get("llm_model")
+        temperature = self.config.get("llm_temperature")
+        custom_llm_provider = self.config.get("llm_custom_provider")
```

Comment:
Let's change `custom_llm_provider` to `llm_custom_provider` for consistency.

Reply:
Nevermind on this one. Realized custom_llm_provider is the name of the lite_llm option and there's no reason to break people's configs by changing the name of our config option.